models/gas/optimism/gas_optimism_fees.sql

```diff
@@ -0,0 +1,51 @@
+{{ config(
```

Comment:
could be cool to add:

(l1_gas_price * gas_used)/1e18 = tx_fee_equivalent_on_l1_native
p.price * (l1_gas_price * gas_used)/1e18 = tx_fee_equivalent_on_l1_usd